DS13/code/game/objects/replicator.dm

```diff
@@ -0,0 +1,322 @@
+/obj/machinery/replicator //Coded by "AbsurdlyLudicrous", tweaked to use voice + cleaned up by Kmc.
+	name = "replicator"
+	desc = "An advanced energy -> matter synthesizer which is charged by <i>biomatter</i> and power. Click it to see the menu and simply say what you want to it."
+	icon = 'DS13/icons/obj/replicator.dmi'
+	icon_state = "replicator"
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 40
+	pixel_y = 32 //So it glues to the wall
```

Comment:
What if i want mine to the south,this is stuff you do in var edits in mapping or in subtypes,not the parent

Reply:
REEEEE

You can do it to the south because of goonwalls

Reply:
See how APC's do their pixel stuff